search/includes/classes/class-health.php

```diff
@@ -322,10 +335,26 @@ public static function validate_index_posts_content( $start_post_id = 1, $last_p
 			}
 		} while ( $start_post_id <= $last_post_id );
 
+		$this->delete_validate_content_ongoing();
+
 		return $results;
 	}
 
-	public static function validate_index_posts_content_batch( $indexable, $start_post_id, $next_batch_post_id, $inspect ) {
+	public function is_validate_content_ongoing(): bool {
+		$is_locked = get_transient( self::CONTENT_VALIDATION_LOCK_NAME, false );
+
+		return (bool) $is_locked;
+	}
+
+	public function reset_validate_content_ongoing() {
```

Comment:
I think this should be `set_validate_content_lock` since all it does is set the transient `vip_search_content_validation_ongoing` to true for 15 minutes on a loop that acts as a lock. 

Reset implies some previous standard state that we're going back to. That isn't the case here.

What do you think?